ambry-store/src/main/java/com.github.ambry.store/BlobStoreStats.java

```diff
@@ -312,6 +319,18 @@ void handleNewDeleteEntry(IndexValue deleteValue, IndexValue originalPutValue) {
     }
   }
 
+  /**
+   * Function that handles new TTL updates after a scan to keep the current {@link ScanResults} relevant.
+   * @param ttlUpdateValue the {@link IndexValue} of the new TTL update
+   * @param originalPutValue the {@link IndexValue} of the original PUT that is getting updated
+   */
+  void handleNewTtlUpdateEntry(IndexValue ttlUpdateValue, IndexValue originalPutValue) {
```

Comment:
+1

Reply:
Yes. Since it is not implemented, I thought I could skip calling but I can add that code just for completeness (so that everything works if this impl is filled out)